app/routes/__index/quest2.tsx

```diff
@@ -0,0 +1,58 @@
+import PlusIconWhite from "~/components/Icons/PlusIcon";
+import { LinkFooter } from "~/components/Buttons/LinkButton";
+import Carousel from "~/components/Quest/Carousel";
+import { db } from "~/utils/db.server";
+import { useLoaderData } from "@remix-run/react";
+import { json } from "@remix-run/node";
+
+
+
+
+export async function loader() {
+    const allProjects = await db.badges.findMany({
+        orderBy: [
+          {
+            token_id: 'desc',
+          }
+        ]});
+
+    return json(allProjects);
+};
+
+
+export default function Quest() {
+    const badges = useLoaderData();
```

Comment:
You should type your data with Badges from Prisma

Reply:
![CleanShot 2022-12-27 at 10 06 37](https://user-images.githubusercontent.com/25151724/209642119-be159456-981f-4de1-95e0-1eb4dfd8adfb.png)
 could you help ? @schwepps

Reply:
Sure. It's because of the type returned by useLoaderData();

We must do something like this:
`const badges = useLoaderData<unknown> as Badges`